content/en/docs/TrueNAS/plugins.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Using Plugins"
+linkTitle: "Using Plugins"
+description: "How to use plugins in TrueNAS CORE and TrueNAS Enterprise"
+---
+
+{{% alert title="Notice" color="info" %}}
+This feature is available in TrueNAS CORE and specially licensed versions of TrueNAS Enterprise.
+For assistance, please contact iX Support:
```

Comment:
This sentence and ambiguous. I've interpreted this sentence as "If you have TrueNAS Core, and you have questions about Plugins, then you can contact iX Support". As it stands right now, we (iX Support team) doesn't support TrueNAS Core plugin installations. I'd recommend this be a little more direct and say something like

"If you have TrueNAS Core and need help, please reach out to the community. If you have TrueNAS Enterprise, please reach out to iX Support"